src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -194,6 +194,18 @@ public int NewLines_Braces_Method
             set { SetBooleanOption(CSharpFormattingOptions.NewLinesForBracesInMethods, value); }
         }
 
+        public int NewLines_Braces_Property
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions.NewLinesForBracesInProperties); }
+            set { SetBooleanOption(CSharpFormattingOptions.NewLinesForBracesInProperties, value); }
+        }
+
+        public int NewLines_Braces_Accessor
```

Comment:
ATM I would suggest using just the Property braces option. It's very easy to let the options explode when you're going the fine grained route. At the end 80% of the options are not used :/.